packages/fxa-settings/src/models/Account.ts

```diff
@@ -897,6 +910,26 @@ export class Account implements AccountData {
     );
   }
 
+  /* TODO: Remove this and use GQL instead. We can't check for verified sessions
+   * unless you've already got one (oof) in at least the PW reset flow due to
+   * sessionToken.mustVerify which was added here: https://github.com/mozilla/fxa/pull/7512
+   * The unverified session token returned by a password reset contains `mustVerify`
+   * which causes the 'Must verify' error to be thrown and a redirect to occur */
+  async isSessionVerifiedAuthClient() {
```

Comment:
I like the idea of some differentiator, like ending in `AuthClient` for methods that use auth client over apollo client. It makes it more clear to me what's happening when we call it and that we need to refactor it.